### ISSUE TITLE
Multiple SAN support

### ISSUE THREAD

**Issue by funzoneq** [schubergphilis/letsencrypt#8]
Hi,

I'd like to be able to use multiple hostnames in my certificate. For example:

```
letsencrypt_certificate 'test.example.com' do
  crt      '/etc/ssl/test.example.com.crt'
  key      '/etc/ssl/test.example.com.key'
  method   'http'
  wwwroot  '/var/www'
  domains ['example.example.com', 'www.example.com', 'example.com']
end
```

See example for generating the X509 cert here: https://gist.github.com/arusso/d5a3195773c2ca3717d4

**Comment by thoutenbos** [comment]
@sawanoboly started working on this in #5

**Comment by thoutenbos** [comment]
Available in v0.1.3
